superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -242,8 +243,12 @@ export function useSingleViewResource<D extends object = any>(
         loading: true,
       });
 
+      const endpoint =
+        path_suffix !== ''
+          ? `/api/v1/${resourceName}/${resourceID}/${path_suffix}`
+          : `/api/v1/${resourceName}/${resourceID}`;
```

Comment:
nit, but maybe something like this may be a bit dryer: 
```
const endpoint = `/api/v1/${resourceName}/${resourceID}${path_suffix ? `/${path_suffix}` :  '' }`
or 
const endpoint = `/api/v1/${resourceName}/${resourceID}${path_suffix && `/${path_suffix}` || ''}`
```